tests/test_decorator.py

```diff
@@ -89,9 +91,9 @@ def test_command_line_1(capfd):
 
 
 expected_out = (
-    "tests/trio_options.py:2:1: TRIO107 "
+    f"{file_path}:5:1: TRIO107 "
     + Visitor107_108.error_codes["TRIO107"].format(
-        "exit", Statement("function definition", 2)
+        "exit", Statement("function definition", 5)
```

Comment:
Update to using the new `file_path` variable instead of hardcoding the expected file location in the error message. See the comment for `trio_options.py` for the row change.